main.go

```diff
@@ -97,11 +97,12 @@ func main() {
 						continue
 					}
 				}else{
+					formattedUrl := url
 					if *format {
-						*url = format_url(*url, *timezone)
+						*formattedUrl = format_url(*url, *timezone)
```

Comment:
this still isn't working properly. I'm thinking perhaps the original URL needs to be copied with each iteration? Still figuring out how golang's pointers work, though